### PR TITLE
New PyMOL command for creating publication quality images about receptor-ligand interactions.

### PR DESCRIPTION
**Pull request by tevang**
I committed a simple script that creates nice visualizations of receptor-ligand interactions for publications. The script adds a new command named "show_ligand_interactions".

### CODE REVIEW

File: show_ligand_interactions.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#show_ligand_interactions v.1.0
+# author: Thomas Evangelidis, 2019
+# License: BSD-2-Clause
+
+from pymol import cmd, util
+
+def show_ligand_interactions(recsel="not hetatm", ligsel="hetatm", cuttof=5):
```
Comment (speleo3) [inline show_ligand_interactions.py]: do you want to rename `cuttof` of `cutoff`? Looks like a typo to me.

File: show_ligand_interactions.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    cutoff = float: show as sticks all receptor residues within this distance from the ligand {default: 5.0}
+    """
+    cmd.select('ligand', ligsel)
+    cmd.select('receptor', recsel)
+    cmd.bg_color('white')
+    cmd.show_as('cartoon')
+    cmd.show_as('sticks', 'hetatm')
+    cmd.set('cartoon_transparency', 0.2)
+    cmd.spectrum(selection=recsel+" or "+ligsel,byres=1)
+    util.cbag('not name C*')
```
Comment (speleo3) [inline show_ligand_interactions.py]: `not elem C` is more robust, e.g. wouldn't include calcium atoms

File: show_ligand_interactions.py (hunk 1 above)
Comment (tevang) [reply show_ligand_interactions.py]: yes, thank you!

File: show_ligand_interactions.py (hunk 2 above)
Comment (tevang) [reply show_ligand_interactions.py]: indeed!

**Pull request merged**